### PR TITLE
Option to not store credentials in file

### PR DESCRIPTION
**Pull request by vikeri**
**Motivation:** I don't like storing credentials in config files and a safer solution is to store them in the Keychain Access app.

Therefore I added a link that describes how one may go about to store the passwords in the Keychain Access app in OSX instead.

Found a [stackoverflow post](http://stackoverflow.com/a/24480579/1836121) but it wasn't very detailed so I decided to write it up myself. I understand if you have some policy against external links to blogs and I guess we could link to the stackoverflow explanation then instead.

### CODE REVIEW

**Comment by vikeri** [comment]
Screenshot

skarmavbild 2016-04-20 kl 16 23 51

**Comment by satya164** [comment]
Could you move it to the end of the section instead?

File: docs/SignedAPKAndroid.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ _Note: Remember to keep your keystore file private and never commit it to versio
 
 ### Setting up gradle variables
 
+_Note: This is a global way that will work for any platform, if you are not keen on storing your passwords in plaintext and you are running OSX, you can also [store your credentials in the Keychain Access app](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/). Then you can skip the two last rows in `~/.gradle/gradle.properties`._
```
Comment (satya164) [inline docs/SignedAPKAndroid.md]: Actually it won't work for Windows. So let's remove the text `This is a global way that will work for any platform`

Comment (vikeri) [inline docs/SignedAPKAndroid.md]: @satya164: Check!

File: docs/SignedAPKAndroid.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ _Note: Remember to keep your keystore file private and never commit it to versio
 
 ### Setting up gradle variables
 
+_Note: If you are not keen on storing your passwords in plaintext and you are running OSX, you can also [store your credentials in the Keychain Access app](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/). Then you can skip the two last rows in `~/.gradle/gradle.properties`._
```
Comment (satya164) [inline docs/SignedAPKAndroid.md]: Can you move it to the bottom of the paragraph please?

Comment (vikeri) [inline docs/SignedAPKAndroid.md]: @satya164 Sure, do you mean after the other note about saving the keystore?

Comment (satya164) [inline docs/SignedAPKAndroid.md]: @vikeri yes

Comment (vikeri) [inline docs/SignedAPKAndroid.md]: @satya164 fixed!

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**